package/cfg/rke2-cis-1.5/master.yaml

```diff
@@ -0,0 +1,1081 @@
+---
+controls:
+version: 1.5
+id: 1
+text: "Master Node Security Configuration"
+type: "master"
+groups:
+- id: 1.1
+  text: "Master Node Configuration Files "
+  checks:
+    - id: 1.1.1
+      text: "Ensure that the API server pod specification file permissions are set to 644 or more restrictive (Scored)"
+      audit: "/bin/sh -c 'if test -e $apiserverconf; then stat -c %a $apiserverconf; fi'"
+      tests:
+        bin_op: or
+        test_items:
+          - flag: "644"
+            compare:
+              op: eq
+              value: "644"
+            set: true
+          - flag: "640"
+            compare:
+              op: eq
+              value: "640"
+            set: true
+          - flag: "600"
+            compare:
+              op: eq
+              value: "600"
+            set: true
+          - flag: "444"
+            compare:
+              op: eq
+              value: "444"
+            set: true
+          - flag: "440"
+            compare:
+              op: eq
+              value: "440"
+            set: true
+          - flag: "400"
+            compare:
+              op: eq
+              value: "400"
+            set: true
+          - flag: "000"
+            compare:
+              op: eq
+              value: "000"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the
+        master node.
+        For example, chmod 644 $apiserverconf
+      scored: true
+
+    - id: 1.1.2
+      text: "Ensure that the API server pod specification file ownership is set to root:root (Scored)"
+      audit: "/bin/sh -c 'if test -e $apiserverconf; then stat -c %U:%G $apiserverconf; fi'"
+      tests:
+        test_items:
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root $apiserverconf
+      scored: true
+
+    - id: 1.1.3
+      text: "Ensure that the controller manager pod specification file permissions are set to 644 or more restrictive (Scored)"
+      audit: "/bin/sh -c 'if test -e $controllermanagerconf; then stat -c %a $controllermanagerconf; fi'"
+      tests:
+        test_items:
```

Comment:
maybe copied from elsewhere, but why does the test for 1.1.1 have 7 different perms it can test for and this only have one? I see the latest iteration for rke1 has this fixed:
https://github.com/rancher/security-scan/blob/master/package/cfg/rke-cis-1.5-hardened/master.yaml#L74